ZEP000-wf-stream-processing.md

```diff
@@ -130,6 +130,8 @@ The processor rejects the command and doesn't apply the business logic if the pr
 
 ![concurrent-commands](ZEP-wf-stream-processing/event-sourcing-concurrent-commands.png)    
 
+> Currently, all workflow instance records are events (except the cancel command). If the preconditions of an event doesn't met then the processor ignors the event and doesn't apply the business logic. From a consumer of the stream, it is not visible that the event was not processed.   
```

Comment:
It's not clear to me what your intent is here. I think that problem would be resolved when this ZEP is implemented. I also don't understand why it is specifically mentioned under concurrent commands. What am I missing?

Reply:
I added this part for comparison with the current behavior. I guess that it is not always clear which behavior changes. Maybe, it's better to list all changes under the section "Compatibility". WDYT?

Reply:
Yes, I think we should find a place to compile all the changes in comparison to the current behaviour. Compatibility sounds like the best spot for this.